flowauth/backend/flowauth/admin.py

```diff
@@ -499,9 +512,9 @@ def edit_group_servers(group_id):
             )
             limits.longest_life = server["max_life"]
         if limits.longest_life > server_obj.longest_token_life:
-            abort(400, "lifetime_too_long")
+            raise InvalidUsage("lifetime_too_long")
```

Comment:
Minor nitpick, but do we want to remove the underscores to be consistent (and slightly more human-readable) with the invocations above? No strong opinion though.

Reply:
Oh definitely - basically all of these need to be changed to `InvalidUsage("Verbose error message", payload={"bad_field":"blah"})` or similar, but I felt like this PR was already getting too big ;)